server/routes/api.js

```diff
@@ -146,53 +146,28 @@ router.post('/register',
       
 });
 
-// Update the role and admin permissions for the member
-router.put('/update/member/', passport.authenticate('jwt', {session: false}), async (req, res) =>
-{
-    const newRole = req.body.role;
-    const newPermission = req.body.permission;
-    const memberID = req.body.memberID;
-
-    const updatedMember = await Member.findOneAndUpdate({_id: memberID}, {role: newRole, admin: newPermission});
-
-    if (updatedMember)
-    {
-        return res.json({success: true});
-    }
-    else
-    {
-        return res.json({success: false});
-    }
-});
-
-// Delete the member
-router.delete('/delete/member/:memberID', passport.authenticate('jwt', {session: false}), async (req, res) =>
-{
-    const memberID = req.params.memberID;
-
-    const deletedMember = await Member.findOneAndDelete({_id: memberID});
-
-    if (deletedMember)
-    {
-        return res.json({success: true});
-    }
-    else
-    {
-        return res.json({success: false});
-    }
-});
-
 
 router.get('/get/events/for/:id', async (req, res) =>
 {
     const id = req.params.id;
     // Initialize the lists where the events will be added
     let events = [];
     let eventIDs = [];
+    let paid = {};
 
-    // Find the IDs of the events that the user is participating in
+    // Find the IDs of the events that the user is interested in or has paid for
     await Member_event.find({member: id})
     .then((docs) =>
+    {
+        docs.forEach(item => {
+            eventIDs.push(item.event);
+            paid[item.event] = item.paid;
```

Comment:
member_event no longer has a `paid` field (we now have a ticket entity for that), so this (and other places) should be changed to not use it.  Whether or not a user has paid for an event should be found out by checking if they have a ticket for it :D